Add instructions to start the app (#16)
micahgodbolt/fluid-react-starter: README.md

npm start
```

> To run our local server, Tinylicious, on the default values of `localhost:7070`, please enter the following into a terminal window:

Reviewer: ```suggestion
To run a local Tinylicious server using the default configuration, enter the following into the terminal:
```

```
npx tinylicous
```

Now, with our local service running in the background, we need to connect the application to it. The app has already been configured to this so now we just need to run the following in a new terminal window to start the app.

```bash
npm i
npm run start
```

> To see how this is working, please take a look at `config.ts` where you will see the following values specified.

Reviewer: ```suggestion
To see how this is working, take a look at `config.ts` where you will see the following values specified.
```
